src/fsharp/FSharp.Core/printf.fs

```diff
@@ -951,7 +948,7 @@ module internal PrintfImpl =
                 arr.[start + i] <- s.Pop()
             arr
         
-        member this.GetArgumentAndTypesAsArrays
+        member __.GetArgumentAndTypesAsArrays
```

Comment:
another reason people tend to replace `this.` with `__.` is when the identifier isn't used it comes up as an unused declaration

![](http://i.imgur.com/yDNEd7v.png)

while `__.` is clean.

![](http://i.imgur.com/ymFD0wg.png) 

If `this`/`self` isn't being used it's just visual noise, like superfluous parentheses.